### PR TITLE
Use Protos

### PR DESCRIPTION
**Pull request by TurtIeSocks**
- For quest_reward_types
- For quest_conditions

### CODE REVIEW

File: master-latest.json [hunk 1]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "POKEMON_ENCOUNTER",
+			"text": "Pokemon Encounter"
+		},
+		"8": {
+			"prototext": "POKECOIN",
+			"text": "Pokecoin"
+		},
+		"9": {
+			"prototext": "XL_CANDY",
+			"text": "Xl Candy"
```
Comment (Mygod) [inline master-latest.json]: XL Candy

File: master-latest.json [hunk 2]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "WITH_UNIQUE_POKESTOP",
+			"text": "With Unique Pokestop"
+		},
+		"13": {
+			"prototext": "WITH_QUEST_CONTEXT",
+			"text": "With Quest Context"
+		},
+		"14": {
+			"prototext": "WITH_THROW_TYPE_IN_A_ROW",
+			"text": "With Throw Type In A Row"
```
Comment (Mygod) [inline master-latest.json:44768]: With Throw Type in a Row

File: master-latest.json [hunk 3]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "WITH_WIN_BATTLE_STATUS",
+			"text": "With Win Battle Status"
+		},
+		"19": {
+			"prototext": "WITH_NEW_FRIEND",
+			"text": "With New Friend"
+		},
+		"20": {
+			"prototext": "WITH_DAYS_IN_A_ROW",
+			"text": "With Days In A Row"
```
Comment (Mygod) [inline master-latest.json:44792]: With Days in a Row

File: master-latest.json [hunk 4]
```diff
@@ -44790,5 +44629,233 @@
...
+		},
+		"21": {
+			"prototext": "WITH_UNIQUE_POKEMON",
+			"text": "With Unique Pokemon"
+		},
+		"22": {
+			"prototext": "WITH_NPC_COMBAT",
+			"text": "With Npc Combat"
```
Comment (Mygod) [inline master-latest.json]: NPC

File: master-latest.json [hunk 5]
```diff
@@ -44790,5 +44629,233 @@
...
+		},
+		"23": {
+			"prototext": "WITH_PVP_COMBAT",
+			"text": "With Pvp Combat"
```
Comment (Mygod) [inline master-latest.json]: PvP

File: master-latest.json [hunk 6]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "WITH_INVASION_CHARACTER",
+			"text": "With Invasion Character"
+		},
+		"28": {
+			"prototext": "WITH_BUDDY",
+			"text": "With Buddy"
+		},
+		"29": {
+			"prototext": "WITH_BUDDY_INTERESTING_POI",
+			"text": "With Buddy Interesting Poi"
```
Comment (Mygod) [inline master-latest.json]: PoI

File: master-latest.json [hunk 7]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "WITH_SINGLE_DAY",
+			"text": "With Single Day"
+		},
+		"33": {
+			"prototext": "WITH_UNIQUE_POKEMON_TEAM",
+			"text": "With Unique Pokemon Team"
+		},
+		"34": {
+			"prototext": "WITH_MAX_CP",
+			"text": "With Max Cp"
```
Comment (Mygod) [inline master-latest.json]: CP

File: master-latest.json [hunk 8]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "WITH_LEGENDARY_POKEMON",
+			"text": "With Legendary Pokemon"
+		},
+		"37": {
+			"prototext": "WITH_TEMP_EVO_POKEMON",
+			"text": "With Temp Evo Pokemon"
+		},
+		"38": {
+			"prototext": "WITH_GBL_RANK",
+			"text": "With Gbl Rank"
```
Comment (Mygod) [inline master-latest.json]: GBL

File: master-latest.json [hunk 9]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "WITH_LUCKY_POKEMON",
+			"text": "With Lucky Pokemon"
+		},
+		"36": {
+			"prototext": "WITH_LEGENDARY_POKEMON",
+			"text": "With Legendary Pokemon"
+		},
+		"37": {
+			"prototext": "WITH_TEMP_EVO_POKEMON",
+			"text": "With Temp Evo Pokemon"
```
Comment (Mygod) [inline master-latest.json:44860]: Temporary Evolution

File: master-latest.json [hunk 10]
```diff
@@ -44790,5 +44629,233 @@
...
+		},
+		"38": {
+			"prototext": "WITH_GBL_RANK",
+			"text": "With Gbl Rank"
+		},
+		"39": {
+			"prototext": "WITH_CATCHES_IN_A_ROW",
+			"text": "With Catches In A Row"
```
Comment (Mygod) [inline master-latest.json:44868]: in a Row

File: master-latest.json [hunk 11]
```diff
@@ -44790,5 +44629,233 @@
...
+			"prototext": "WITH_ENCOUNTER_TYPE",
+			"text": "With Encounter Type"
+		},
+		"41": {
+			"prototext": "WITH_COMBAT_TYPE",
+			"text": "With Combat Type"
+		},
+		"42": {
+			"prototext": "WITH_GEOTARGETED_POI",
+			"text": "With Geotargeted Poi"
```
Comment (Mygod) [inline master-latest.json]: PoI

**Review by Mygod — COMMENTED** [review]
I think these should be manually translated instead of auto generated, maybe like PMSF?

**Comment by Fabio1988** [comment]
I don't think that those enum values are meaningful enough for the Players? How to translate in different languages? :)

**Pull request merged**